### ISSUE TITLE
Does Geospatial API have support on tablets without built-in SIM if it still has hotspot internet access?

### ISSUE THREAD

**Issue by murphysurveys** [google-ar/arcore-unity-extensions#168]
Does Geospatial API have support on tablets without built-in SIM if it still has hotspot internet access?

I looked at the ARCore device list but it doesn't say explicitly if the wi-fi only version of an ARCore tablet such as the Samsung Galaxy Tab A8 will work with the Geospatial API assuming I provide mobile internet from phone hotspot.

**Comment by devbridie** [comment]
Yes, Geospatial API should be fine as long as there's device support and internet access!
